ops/ansible/playbooks-ccs/data_server_collect_flame_graph.yml

```diff
@@ -49,6 +48,7 @@
           - cmake
           - gcc
           - gcc-c++
+          - java-1.8.0-openjdk-devel
```

Comment:
We should consider updating our default builds to install the JDK like this, rather than the JRE. It's super useful to have a JDK around for all sorts of things.

Reply:
I'm pretty sure we already do: https://github.com/CMSgov/beneficiary-fhir-data/blob/a5e50a508c7df81fadb82f8736994d94adc4f6be/ops/ansible/roles/bfd-server/tasks/install.yml#L9